sync/invoice_category.go

```diff
@@ -93,7 +100,15 @@ func ToInvoiceCategory(category entity.Category) (model.InvoiceCategory, error)
 		if len(arr) < 2 {
 			return model.InvoiceCategory{}, fmt.Errorf("cannot parse source: %s: expected format `cluster:namespace`", category.Source)
 		}
-		name = fmt.Sprintf("Zone: %s - Namespace: %s", arr[0], arr[1])
+		zone := arr[0]
+		if m != nil {
+			mapped, err := m.MapZoneName(context.Background(), zone)
+			if err != nil {
+				return model.InvoiceCategory{}, fmt.Errorf("error mapping zone source %q to name: %w", zone, err)
+			}
+			zone = mapped
+		}
```

Comment:
Alternative idea might be to just skip the reconciliation if there's no zone mapping instead of aborting the whole process. For example, there might be a new Zone set up but not configured in the mapping. Should that really abort the remaining categories to be synced even though they'd have valid mappings?

Reply:
Totally agree!

We would need to remodel the control flow (a bit). So I'd like to do that in a follow up.